frontend/src/components/Menu/Menu.js

```diff
@@ -21,6 +20,9 @@ const Menu = ({open}) => {
                 <a>
                     <span aria-label="Offers Made">Offers Made</span>
                 </a>
+                <a href="/newInquiry">
```

Comment:
if you are using react router you may want to use a `<Link>` instead
you can also use them in styled component context 
```

const StyledComponent = styled(Link)`
  ...
`
```

Reply:
Yes, definitely something I will add in the future! Thanks for the input Lisa!